### DISCUSSION TITLE
Number of Action's installs

### DISCUSSION THREAD

**Discussion by ggfmarques** [community/community#35319] [Actions] [answered]
Hi,

I am interested in publishing an Action in [Github Actions Marketplace](https://github.com/marketplace?type=actions) and in the future I would like to understand if people are using it a lot or not.
I would expect something similar to what we can see for the Apps (number of installs):
![Screenshot 2022-10-06 at 11 01 10](https://user-images.githubusercontent.com/115146355/194285413-59019a06-c03c-472d-94c1-17e20150abb4.png)

But from what I see, no "Installs" information is available in the Marketplace for the Actions.
![Screenshot 2022-10-06 at 11 07 30](https://user-images.githubusercontent.com/115146355/194286558-4c75696f-f3df-455e-ab02-86774b3c8c75.png)

But will I still be able to get such information for an Action published by me?
If yes, how can I obtain it?

Thanks

**Comment by spenserblack** [comment, marked as answer]
GitHub Actions that you create will appear in dependency graphs. You can get a count that way. Here's an example: https://github.com/spenserblack/actions-tag-to-release/network/dependents

**Comment by jrfnl** [comment]
Came here with the same question, but this does not appear to be true in all cases....

Case in point: https://github.com/PHPCSStandards/xmllint-validate/network/dependents

And yes, I know for a fact it is being used as I added it into the workflows of a number of repos I manage. I'm just curious to see if other people have started using it too (or not), but can't seem to find out.
